src/hooks/core/chat/useChatUtilsHook.ts

```diff
@@ -6,15 +6,14 @@ import {CommonActions, useNavigation} from '@react-navigation/native';
 import {atom, useRecoilState} from 'recoil';
 
 import AnonymousMessageRepo from '../../../service/repo/anonymousMessageRepo';
+import ChannelList from '../../../database/schema/ChannelListSchema';
 import SignedMessageRepo from '../../../service/repo/signedMessageRepo';
 import UseChatUtilsHook from '../../../../types/hooks/screens/useChatUtilsHook.types';
 import UserSchema from '../../../database/schema/UserSchema';
 import useLocalDatabaseHook from '../../../database/hooks/useLocalDatabaseHook';
+import useUserAuthHook from '../auth/useUserAuthHook';
 import {ANON_PM, GROUP_INFO} from '../constant';
-import {
-  BetterSocialChannelType,
-  ChannelList
-} from '../../../../types/database/schema/ChannelList.types';
+import {BetterSocialChannelType} from '../../../../types/database/schema/ChannelList.types';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `ChannelList` has been moved from the grouped import statement to a separate line. This change doesn't seem to have any functional impact, but it does break the consistency of the import statements. It would be better to keep all imports from the same module together for readability and maintainability.
```diff
- import ChannelList from '../../../database/schema/ChannelListSchema';
- import {BetterSocialChannelType} from '../../../../types/database/schema/ChannelList.types';
+ import {
+   BetterSocialChannelType,
+   ChannelList
+ } from '../../../../types/database/schema/ChannelList.types';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->